tests/phpunit/RealDataTest.php

```diff
@@ -24,7 +28,8 @@ public function testProcess()
             );
 
             // compare column counts
-            $parsedFile = file($table->getPathname());
+            $headerCount = null;
+            $parsedFile = new CsvFile($table->getPathname());
```

Comment:
Tento test bol velmi divny. Ma porovnavat, ze v kazdom riadku CSV je rovnaky pocet riadkov.
No nacitava to kazdy riadok ako string (vysledok `file` funkcie).... a potom robi `count(string)` co vracia stale 1, bez ohladu na to co je v tom stringu. Preto som to fixol.